src/Sylius/Bundle/WebBundle/Resources/views/Backend/Inventory/macros.html.twig

```diff
@@ -72,7 +72,7 @@
                                     <td>
                                         <ul>
                                         {% for option in variant.options %}
-                                            <li><strong>{{ option.name }}</strong>: {{ option.value }}</li>
+                                            <li><strong>{{ option.presentation }}</strong>: {{ option.value }}</li>
```

Comment:
Agree with all :+1: 

**Notice:** There is on-going effort on moving to the new admin with grids, so if it's just simple rename then fine, but I wouldn't invest much time in current templates since they will be removed soon :wink: